jojo.py

```diff
@@ -107,6 +107,172 @@ def open_json(json_file):
     return df
 
 
+def summon_stand_user_info(mcr):
+    '''
+    スタンド能力と使用者を紐づけるアマスタを生成します。
+    外部ファイルの設定を優先とし、既に生成されている場合はスキップします。
+
+    Parameter
+        mcr : MCRcon
+            Rconのサーバ情報
+    '''
+    str_dir = 'json_list'
+    str_stand_file = 'stand_list.json'
+    entity_name = 'minecraft:armor_stand'
+    X = 0
+    Y = -74
+    Z = 0
+    invulnerable = True
+    gravity = False
+    contnts = open_json(f'{str_dir}/{str_stand_file}')
+
+    for key in contnts.keys():
+        
+        #ワールドのエンティティの情報を取得
+        resp = get_entity_data(mcr, entity_name, None, key, 'Tags')
+        
+        #ワールドのエンティティが存在しない場合
+        if resp is None:
+            #エンティティを新規で生成する
+            _ = set_entity_data(mcr, entity_name, X, Y, Z, invulnerable, gravity, resp[0], key)

```

Comment:
respはlistではない気がします。
引数にはrespだけで良いと思います。

Reply:
それから返り値ように変数を用意しなくても、関数内のreturnは実行されるはずなので、以下のような記述でよいです。

```suggestion
            set_entity_data(mcr, entity_name, X, Y, Z, invulnerable, gravity, resp, key)
```

Reply:
resp[0]としていた箇所をstand_list.jsonのValueになるように修正しました。